jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListViewImpl.java

```diff
@@ -429,32 +428,42 @@ public ProcessInstanceSummary getValue(ProcessInstanceSummary process) {
     private Column<ProcessInstanceSummary, ProcessInstanceSummary> initActionsColumn() {
         List<HasCell<ProcessInstanceSummary, ?>> cells = new LinkedList<HasCell<ProcessInstanceSummary, ?>>();
 
-        cells.add(new ProcessInstanceSummaryActionCell(constants.Signal(), new Delegate<ProcessInstanceSummary>() {
-            @Override
-            public void execute( final ProcessInstanceSummary processInstance ) {
+        cells.add(new ProcessInstanceSummaryActionCell(
+            constants.Signal(),
+            ProcessInstanceSummaryActionCell.PREDICATE_STATE_ACTIVE,
+            processInstance -> {
                 presenter.signalProcessInstance(processInstance);
             }
-        } ) );
-
-        cells.add(new ProcessInstanceSummaryActionCell(constants.Abort(), new Delegate<ProcessInstanceSummary>() {
-            @Override
-            public void execute( ProcessInstanceSummary processInstance ) {
-                if ( Window.confirm( constants.Abort_Process_Instance() ) ) {
-                    presenter.abortProcessInstance( processInstance.getDeploymentId(), processInstance.getProcessInstanceId() );
+        ));
+
+        cells.add(new ProcessInstanceSummaryActionCell(
+            constants.Abort(),
+            ProcessInstanceSummaryActionCell.PREDICATE_STATE_ACTIVE,
+            processInstance -> {
+                if(Window.confirm(constants.Abort_Process_Instance())){
+                    presenter.abortProcessInstance(processInstance.getDeploymentId(), processInstance.getProcessInstanceId());
                 }
             }
-        } ) );
+        ));
 
-        CompositeCell<ProcessInstanceSummary> cell = new CompositeCell<ProcessInstanceSummary>( cells );
-        Column<ProcessInstanceSummary, ProcessInstanceSummary> actionsColumn = new Column<ProcessInstanceSummary, ProcessInstanceSummary>( cell ) {
+        cells.add(new ProcessInstanceSummaryActionCell(
+            constants.ViewJobs(),
+            ProcessInstanceSummaryActionCell.PREDICATE_TRUE,
```

Comment:
@cristianonicolai Any recommendation about how to implement the condition so the "View jobs" button only is displayed if the process instance has jobs? Should I use a dataset lookup to check the jobs associated to each process instance?